packages/cycle-scripts/template/config/javascript/flavor.js

```diff
@@ -0,0 +1,24 @@
+const dependencies = []
```

Comment:
My rationaled on why I think they are part of template sources:  Ones are precompiled template functions, the other are template strings (replacement), together they form the template sources.